src/lib/viewers/controls/settings/SettingsDropdown.tsx

```diff
@@ -74,52 +71,69 @@ export default function SettingsDropdown<V extends Value = string>({
 
     useClickOutside(dropdownElRef, () => setIsOpen(false));
 
+    // Customize the forwarded ref to combine usage with the ref internal to this component
+    React.useImperativeHandle(
+        ref,
+        () => ({
+            focus: (): void => {
+                if (buttonElRef.current) {
+                    buttonElRef.current.focus();
+                }
+            },
+        }),
+        [],
+    );
+
     return (
-        <div ref={dropdownElRef} className={classNames('bp-SettingsDropdown', className)}>
+        <div className={classNames('bp-SettingsDropdown', className)}>
```

Comment:
Moved the ref to a separate container on L92 so that clicking on the label is a valid way to close the dropdown